From 423aae98e0bf9f6fcf29e30e433d9abab6f1f144 Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Mon, 8 Jul 2024 11:55:30 +0200
Subject: [PATCH] board: sr_rzg2l: enable passing dram info from BL3 to u-boot
 in args2&3

Enable passing dram info from bl3 to u-boot using arguments 2 and 3,
allowing u-boot to configure correct dram size.

The standard method using a device-tree blob should be used at a future
point once u-boot parsing issues are resolved, e.g. by using SPL.

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 plat/renesas/rz/board/sr_rzg2l/rz_board.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/renesas/rz/board/sr_rzg2l/rz_board.mk b/plat/renesas/rz/board/sr_rzg2l/rz_board.mk
index 696dc88f7..7fee10c61 100644
--- a/plat/renesas/rz/board/sr_rzg2l/rz_board.mk
+++ b/plat/renesas/rz/board/sr_rzg2l/rz_board.mk
@@ -23,6 +23,6 @@ BL33_ARG1_FDTBLOB := 0
 $(eval $(call add_define,BL33_ARG1_FDTBLOB))
 include lib/libfdt/libfdt.mk
 
-BL33_ARG23_DRAM_INFO := 0
+BL33_ARG23_DRAM_INFO := 1
 $(eval $(call add_define,BL33_ARG23_DRAM_INFO))
 include lib/libfdt/libfdt.mk
-- 
2.35.3

